cli_ui/__init__.py

```diff
@@ -75,6 +71,13 @@ def _setup(**kwargs: ConfigValue) -> None:
         CONFIG[key] = value
 
 
+def _init_coloram() -> None:
```

Comment:
colorama?

Reply:
yeah - windows works really fine until the keyboard bl

Reply:
I mean there's a typo. Let me try the new suggestion feature of github.

```suggestion
def _init_colorama() -> None:
```

Reply:
Oh you fixed it :)